index.js

```diff
@@ -0,0 +1,87 @@
+const http = require("http");
```

Comment:
We recommend using the [template](https://github.com/HackYourFuture-CPH/hyf-project-template) provided at https://github.com/HackYourFuture-CPH/node.js/tree/main/week1/homework.

It includes express and knex, for example, so you can use [query builder](https://knexjs.org/guide/schema-builder.html) instead of raw queries also the next homework will build on these technologies.